components/feature/search/src/main/assets/search/list.json

```diff
@@ -210,7 +224,7 @@
       },
       "experimental-hidden": {
         "visibleDefaultEngines": [
-          "amazon-ca", "amazon-au", "google-2018", "duckduckgo", "google"
+          "google-2018", "duckduckgo", "google"
```

Comment:
What's up with this `experimental-hidden` change?

Reply:
When I was working on Amazon I added these to their region/locale of en-CA and en-AU.  Just noticed that I didn't clean these up.

Reply:
It might be worth double-checking if the `experimental-hidden` is actually used on mobile, and if not just removing all of them. I believe desktop used it alongside the ABSearch set-up, and I suspect Mobile doesn't need it.

Reply:
Sure, I'll investigate and if it's not in use I'll remove them. Thanks,

Reply:
Yeah, to my knowledge we do not use this section.